src/schoology-extractor/README.md

```diff
@@ -3,6 +3,12 @@
 This script retrieves and writes out to CSV all students, sections, assignments,
 and submissions for the desired grading period(s).
 
+## Notes on Working With Schoology Data
+
+* The Schoology API handles _negative attendance_ events: if a student is marked
+  as present, or is not marked at all, then the system will not return a record
+  for that day.
```

Comment:
I was surprised to realize this point and this felt like the best way to keep it in mind.